languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,7 +34110,7 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
-    <rulegroup id="ESPACE_APRES_POINT" name="Vérifiez l'espace après le point.">
+      <rulegroup id="ESPACE_APRES_POINT" name="Vérifiez l'espace après le point.">
```

Comment:
How did that space get in here? Now the rulegroup has the same indent as the AP below...